node/src/main/scala/org/bitcoins/node/networking/peer/DataMessageHandler.scala

```diff
@@ -13,16 +14,17 @@ import scala.concurrent.ExecutionContext
   * that a peer to sent to us on the p2p network, for instance, if we a receive a
   * [[HeadersMessage]] we should store those headers in our database
   */
-class DataMessageHandler(dbConfig: DbConfig)(implicit ec: ExecutionContext)
+class DataMessageHandler(np: NetworkParameters, dbConfig: DbConfig)(
+    implicit ec: ExecutionContext)
     extends BitcoinSLogger {
 
-  private val blockHeaderDAO = BlockHeaderDAO(dbConfig)
+  private val blockHeaderDAO = BlockHeaderDAO(np.chainParams, dbConfig)
 
   def handleDataPayload(
       payload: DataPayload,
       peerMsgSender: PeerMessageSender): Unit = payload match {
     case headersMsg: HeadersMessage =>
-      blockHeaderDAO.upsertAll(headersMsg.headers.toVector)
+    //blockHeaderDAO.upsertAll(headersMsg.headers.toVector)
```

Comment:
ping

Reply:
i'm going to leave this uncommented for now, and try to contain as many changes in this PR as possible to the chain project.